sematic/external_resource.py

```diff
@@ -96,6 +96,26 @@ def is_terminal(self) -> bool:
 }
 
 
+@unique
+class ManagedBy(Enum):
+    """Represents what entity is managing the state of the resource
+
+    Attributes
+    ----------
+    LOCAL:
+        The resource's state is being managed locally.
+    REMOTE:
+        The resource's state is being managed by the server.
```

Comment:
I believe `RESOLVER` and `SERVER` would be better fitting names.

Reply:
changed as requested